lib/generamba/cli/gen_command.rb

```diff
@@ -58,5 +61,24 @@ def gen(module_name, template_name)
       generator.generate_module(module_name, code_module, template)
     end
 
+    no_commands {
+      def print_parameters_in_table(parameters, module_name)
+        params = parameters.clone
+
+        templates = []
+
+        params["templates"].each do |param|
```

Comment:
@etolstoy We have a problem. With the conversion of the array with strings there is no problem, but with the conversion of the array with hash there are some problems. Example, Rambafile has parameter: 
```
templates:
- {name: MyTemplate1, local: 'path/to/local/template1'}
- {name: MyTemplate2, local: 'path/to/local/template2'}
```
 is a array of hash:
```
{ templates => [ {name => 'MyTemplate1', local => 'path/to/local/template1'}, {name => 'MyTemplate2', local => 'path/to/local/template2'} ] }
```
and that after conversion should look like this:
```
{templates => "MyTemplate1, MyTemplate2"}
```
i.e. we need only `name` parameter from hash. Still, if move this logic in `PrintTable` class, it would be wrong. What do you think?